internal/runners/show/show.go

```diff
@@ -202,15 +215,14 @@ func scriptsData(project *projectfile.Project, conditional *constraints.Conditio
 func platformsData(owner, project string, branchID strfmt.UUID) ([]string, error) {
 	remotePlatforms, fail := model.FetchPlatformsForCommit(branchID)
 	if fail != nil {
-		return nil, locale.WrapError(fail.ToError(), "err_show_get_platforms", "Could not get platform details for commit: {{.V0}}", branchID.String())
+		return nil, locale.WrapError(fail, "err_show_get_platforms", "Could not get platform details for commit: {{.V0}}", branchID.String())
 	}
 
-	platforms := make([]string, len(remotePlatforms))
-	for i, plat := range remotePlatforms {
-		if plat.Kernel == nil || plat.Kernel.Name == nil {
-			continue
-		}
-		platforms[i] = *plat.Kernel.Name
+	plats := platforms.MakePlatformsFromModelPlatforms(remotePlatforms)
+
+	platforms := make([]string, len(plats))
+	for i, plat := range plats {
+		platforms[i] = fmt.Sprintf("%s %s %s-bit", plat.Name, plat.Version, plat.BitWidth)
```

Comment:
Based on your response, I'm confused why we're not using DisplayName, your description makes that sound like exactly the thing we'd want.

Reply:
I hesitated to use it because I thought it might be too much information. But based on our discussions earlier re: platforms it's probably best that we just display with the API returns to us. This will mean outputs like:

```
Platforms:
 - macOS 10.12.6, Darwin 16.6.0, libSystem 1238.60.2 x86 64-bit
 - Windows 10 Version 1803, Windows 10.0.17134.1 x86 64-bit
 - CentOS 6.9, Linux 4.9.125, glibc 2.12 x86 64-bit
```

Reply:
It's a bit verbose but at least if all clients use that then we have the ability to tweak it in a single location.